packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -12,6 +15,9 @@ export interface IExample {
 
   /** JS String for codepen of the example */
   codepenJS?: string;
+
+  /** Custom styles. Partial version of `IExampleCardProps['styles']`. */
+  styles?: IStyleFunctionOrObject<{ theme?: ITheme }, { root: IStyle }>;
 }
```

Comment:
I'm kind of surprised adding the types here is all that is necessary (no change needed in DocPage.base.tsx or something)

Reply:
This type is actually a smaller version of `IExampleCardProps` (we can't depend on `IExampleCardProps` in OUFR because it would cause a circular dep). I looked at the places that create ExampleCards and they just past most of the props straight through, which is why this worked.